Optionally/Options.cs

```diff
@@ -34,6 +34,15 @@ public interface IOption<T>
         /// <param name="filter">Predicate used to test Option</param>
         /// <returns>If Option is Some and the value fulfills the filter, then Some. Otherwise None</returns>
         IOption<T> Where(Func<T, bool> filter);
+
+        /// <summary>
+        /// Performs a Function on the current Option
+        /// </summary>
+        /// <typeparam name="U">Type to convert to</typeparam>
+        /// <param name="ifNone">Function to call if Option is None</param>
+        /// <param name="ifSome">Function to call if Option is None</param>
```

Comment:
For ifSome, probably should read _Option is Some_ instead of _Option is None_

Reply:
Changed and committed